python/pyspark/sql/column.py

```diff
@@ -514,6 +514,17 @@ def isin(self, *cols):
     desc_nulls_first = ignore_unicode_prefix(_unary_op("desc_nulls_first", _desc_nulls_first_doc))
     desc_nulls_last = ignore_unicode_prefix(_unary_op("desc_nulls_last", _desc_nulls_last_doc))
 
+    _isInf_doc = """
+    True if the current expression is inf.
+
+    >>> from pyspark.sql import Row
+    >>> df = spark.createDataFrame([
+            Row(name=u'Tom', height=80.0),
+            Row(name=u'Alice', height=float('inf'))
```

Comment:
dots are required as written in https://github.com/apache/spark/pull/21482#discussion_r197626112

Reply:
Thanks!